.brackets.json

```diff
@@ -0,0 +1,4 @@
+{
+    "spaceUnits": 4,
```

Comment:
I think I added this to help keep a basic consistent code style across the files. I'll pull it and submit a new PR with it and an equivalent .editorconfig file.